openedx/core/djangoapps/content/block_structure/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+This code exists in openedx/core/djangoapp because it needs access to django signalling mechanisms
```

Comment:
s/signalling/signaling - unless you want to explicitly use British spelling for some reason.